tests/Unit/App/Order/Calculator/General/LabelDescriptionCalculatorTest.php

```diff
@@ -84,6 +88,6 @@ function createOrder(string $labelDescription): PdkOrder
         ],
         'order with multiple label description placeholders' => [
```

Comment:
ik zou deze test ff in tweeën (of drieën) splitten, zodat je weet dat alle properties doorkomen én nogmaals de check dat het idd afgeknipt wordt